Atomic/trust.py

```diff
@@ -486,17 +487,20 @@ def get_gpg_id(self, keys):
         """
         if not keys:
             return ""
-        keylist = None
+        (keylist, tmpkey) = None, None
         for key in keys:
             if not os.path.exists(key):
-                # extend when we support parsing inline 'keyData' base64 encoded key
-                return ""
-            else:
-                cmd = ["gpg2", "--with-colons", key]
+                with tempfile.NamedTemporaryFile(delete=False, dir="/run/") as tmpkey:
```

Comment:
`delete=True` (the default) deletes the file when the with-open loop exists, so it's gone when we need it failing the gpg2 cmd.